card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -174,6 +175,9 @@ class DefaultCardDelegate(
     private fun initializeAnalytics(coroutineScope: CoroutineScope) {
         adyenLog(AdyenLogLevel.VERBOSE) { "initializeAnalytics" }
         analyticsManager.initialize(this, coroutineScope)
+
+        val event = GenericEvents.rendered(paymentMethod.type.orEmpty())
```

Comment:
it is unlikely but is it okay to send an empty string for the component param?

Reply:
Indeed it is quite unlikely, but `paymentMethod.type` is nullable and to not add the check we decided to go with `.orEmpty()` approach.
We will monitor the events on data side and check later to see if there are any empty `component`.